wake.spec.in

```diff
@@ -6,9 +6,9 @@ License:       Apache-2; MIT; TFL; WTFPL; CC0
 URL:           https://github.com/sifive/wake
 Vendor:        SiFive, Inc.
 Source0:       https://github.com/sifive/wake/releases/%{name}_%{version}.tar.xz
-Requires:      fuse dash squashfuse iproute
+Requires:      fuse fuse3 dash iproute libzstd lz4 xz lzo zlib
```

Comment:
what is the demand for fuse3 again?

Reply:
squashfuse doesnt use fuse2

Reply:
ok maybe later you and/or I can look at getting rid of fuse2 in wake then